db/postgres/bookings.go

```diff
@@ -140,28 +140,29 @@ func (p PostgresDBStore) CreateBooking(booking *model.Booking) (string, error) {
 	return id, nil
 }
 
-func (p PostgresDBStore) UpdateBooking(id string, booking *model.Booking) error {
+func (p PostgresDBStore) UpdateBooking(id string, booking *model.Booking) (string,error) {
```

Comment:
u could potentially modify the booking input since its a pointer, it will be passed back